.github/workflows/compiletime-crash-tests-pipeline.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Compiletime Crash Tests Pipeline
+
+on: [push]
```

Comment:
Same comment - do we want to run this only on PRs? https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request

otherwise we'll burn CI minutes whenever we push to any branch